docs/backup-restore.md

```diff
@@ -1,32 +1,85 @@
 This document sets out some guidelines for backing up and restoring the databases that Scalar DB supports.
 
-## Cassandra
+## Create Backup
 
-Since Cassandra has a built-in replication mechanism, we don't always need a transactionally-consistent backup to recover a Cassandra cluster.
+### For Transactional Databases
 
-For example, if replication is properly set to 3 and only the data of one of the nodes in a cluster is lost, we don't need a transactionally-consistent backup because the node can be recovered with a normal (transactionally-inconsistent) snapshot and the repair mechanism.
-However, if the quorum of nodes of a cluster loses their data, we need a transactionally-consistent backup to restore the cluster to a certain transactionally-consistent point.
-
-The easiest way to take a transactionally-consistent backup for Scalar DB on Cassandra is to stop a cluster, take the snapshots of all the nodes of the cluster, and start the cluster. If you implement [scalar-admin](https://github.com/scalar-labs/scalar-admin) interface properly in your application, you can easily pause the application without losing on-going transactions.
-
-To minimize mistakes when doing backup and restore operations, it is recommended to use [Cassy](https://github.com/scalar-labs/cassy).
-Cassy is also integrated with `scalar-admin` so it can issue a pause request to the application of a Cassandra cluster.
-Please see [the doc](https://github.com/scalar-labs/cassy/blob/master/docs/getting-started.md#take-cluster-wide-consistent-backups) for more details.
-
-## JDBC databases
+#### JDBC databases
 
 You can take a backup with your favorite way for JDBC databases.
 One requirement for backup in Scalar DB on JDBC databases is that backups for all the Scalar DB managed tables (including the coordinator table) need to be transactionally-consistent or automatically recoverable to a transactionally-consistent state.
 That means that you need to create a consistent snapshot by dumping all tables in a single transaction.
 For example, you can use `mysqldump` command with `--single-transaction` option in MySQL and `pg_dump` command in PostgreSQL to achieve that.
 Or when you use Amazon RDS (Relational Database Service) or Azure Database for MySQL/PostgreSQL, you can restore to any point within the backup retention period with the automated backup feature, which satisfies the requirement.
 
-## Scalar DB server
+### For Non-transactional Databases
+
+#### General way to create a transactionally-consistent backup
+
+The best way to take a transactionally-consistent backup for Scalar DB on a non-transactional database is to use the [Scalar DB server](https://github.com/scalar-labs/scalardb/tree/master/server) or implement the [scalar-admin](https://github.com/scalar-labs/scalar-admin) interface properly in your application, you can easily pause the application without losing ongoing transactions.
```

Comment:
This block and the next block should be placed in a reversed order since this explains the specific way, on the other hand, the next explains the basic strategy.

Reply:
@feeblefakie san,
I think it has to be first.
This section explains general details for creating the transactionally-consistent backup. The next section explains the database-specific procedures for creating backups. I think the user needs to know general procedures to create a Scalar DB backup before dealing with a database-specific.

I think it's better because this section is used in the database-specific procedure. Could you please confirm my understanding?

Reply:
Document revised based on review comments